recipes/mapdamage2/build.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+# seqtk is bundled with mapdamage. The path to seqtk is hardcoded in
+# the mapdamage script so simply adding a dependency on seqtk won't
```

Comment:
I have added two patches, one for the Makefile where the seqtk rule is replaced by a dummy output, one in composition.py in which the subprocess call now makes use of biocondas seqtk. I've been on a long hiatus and wasn't aware of the patch function, neat!